### PR TITLE
Media v2 final touches

### PR DESCRIPTION
**Pull request by sam-prais**

### CODE REVIEW

**Comment by sam-prais** [comment]
Linked to https://github.com/wix-private/media-manager-node-services/pull/505 in media-manager-node-services repo

Jira - https://jira.wixpress.com/browse/APIDOCS-912

**Comment by sam-prais** [comment]
sdk-maual-docs PR - https://github.com/wix-private/sdk-manual-docs/pull/52

File: wix-media-v2/wix-media-v2/guides/media-intro.md [hunk 1]
```diff
@@ -43,10 +43,21 @@ import wixMedia from 'wix-media.v2';
...
+      >**Note:** The `VISITOR_UPLOADS_ROOT` folder is located in the 'Site Files' tab in the UI. However, it is its own root folder and doesn't return when searching in the `MEDIA_ROOT` folder.
+  
+    >**Note:** The `MEDIA_ROOT` system folder is different from the `media-root` folder of the Media Manager. The `MEDIA_ROOT` system folder contains all media in all levels of the 'Site Files' tab in the UI, while the `media-root` folder of the Media Manager contains only the media in the root level of the 'Site Files' tab in the UI. Similar relationships exist between `TRASH_ROOT` and `trash-root`, and `VISITOR_UPLOADS_ROOT` and `visitor-uploads`.
+
+  * **Virtual Folders:** Special categories of folders in the Media Manager that are hidden from the UI and can't be deleted.
+    * `MOBILE_UPLOADS`: Contains all media uploaded using mobile devices. 
+    * `PURCHASED_ITEMS`: Contains all media imported using a purchase flow.
+
+  * **Other Folders:**
```
Comment (emmagrob) [inline wix-media-v2/wix-media-v2/guides/media-intro.md:61]: spacing above this looks off in the preview

File: wix-media-v2/wix-media-v2/Files/guides/files-intro.md [hunk 2]
```diff
@@ -56,34 +56,4 @@ function:
...
-  * Image
-  * Video
-  * Audio
-  * Document
-  * Vector
-  * Archive
-  * Model 3D
-  Learn more about [media file types and size limits](https://support.wix.com/en/article/wix-media-supported-media-file-types-and-file-sizes).
\ No newline at end of file
+</blockquote>
```
[the remark this answers is not in the document]
Comment (sam-prais) [reply wix-media-v2/wix-media-v2/Files/guides/files-intro.md:59]: Yes, was either moved or already existed somewhere else

File: wix-media-v2/wix-media-v2/guides/file-and-folder-ids.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+title: File and Folder IDs
```
[the remark this answers is not in the document]
Comment (sam-prais) [reply wix-media-v2/wix-media-v2/guides/file-and-folder-ids.md:2]: I agree, but it was a whole thing about it needing a separate article based off a user complaint. I'll move in the menu when I publish (is it possible to do so beforehand?)

[the remark this answers is not in the document]
Comment (emmagrob) [reply wix-media-v2/wix-media-v2/guides/file-and-folder-ids.md:2]: you can do it when you publish, just drag it

**Review by emmagrob — APPROVED** [review]
Remember that merging to master is how you publish and that needs to be done in the backstage. 

If you want to see the changes in a preview, merge to your autodocs branch :)

**Pull request merged**